templates/products/new-product.php

```diff
@@ -265,7 +265,17 @@
 
                             <div class="dokan-form-group dokan-right">
                                 <?php wp_nonce_field( 'dokan_add_new_product', 'dokan_add_new_product_nonce' ); ?>
+                                <?php
+                                $display_create_and_add_new_button = true;
+                                if ( method_exists( '\DokanPro\Modules\Subscription\Helper', 'get_vendor_remaining_products' )  ) {
```

Comment:
don't check with method exists, follow below code to check if a module is active or not
```
function_exists( 'dokan_pro' ) && dokan_pro()->module->is_active( 'product_subscription' )
```

Reply:
Changed chacking as you instructed.